code/modules/reagents/newchem/pyro.dm

```diff
@@ -417,6 +417,10 @@
 	for(var/turf/simulated/turf in range(min(created_volume/10,4),T))
 		new /obj/effect/hotspot(turf)
 
+/datum/reagent/phlogiston/reaction_mob(mob/living/M, method=TOUCH, volume)
+	M.IgniteMob()
+	..()
+
 /datum/reagent/phlogiston/on_mob_life(mob/living/M)
 	M.adjust_fire_stacks(1)
 	var/burndmg = max(0.3*M.fire_stacks, 0.3)
```

Comment:
Thanks for restoring the ignite. What would you think of getting rid of the burn damage and instead adding M.adjust_fire_stacks to reaction_mob based on volume?